aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -197,17 +198,22 @@ pub struct TransactionGen<V: Into<Vec<u8>> + Arbitrary + Clone + Debug + Eq + 's
 /// counter value. Each execution of the transaction increments the incarnation counter, and its
 /// value determines the index for choosing the read & write sets of the particular execution.
 #[derive(Debug, Clone)]
-pub enum Transaction<K, V> {
+pub(crate) enum MockTransaction<K, V> {
```

Comment:
Doc string still has `A naive transaction ...`. Should we use mock to be consistent? I think you changed it in couple of places.

Reply:
good catch!